### PR TITLE
[Merged by Bors] - Change some error logs to warnings

### LINKED ISSUE #2633: Only critical logs should use ERROR logging level
## Description

Currently, some logged errors are actually not errors, but debugging messages and warnings. Therefore, there're too many errors in Kibana which makes finding real issues more difficult. Also, it's not clear what cases need to be investigated and what don't. 

We should use the ERROR logging level only for critical errors which require immediate investigation and which mean that something went wrong. Other logs should be classified as WARN/INFO/DEBUG.

After only critical errors are classified to the ERROR level, we may consider using notifications for such logs.

### PR DESCRIPTION
**Pull request by nkryuchkov**
## Motivation

Closes #2633

Some error logs are actually not critical and distract when logs are being watched. This PR changes their levels to WARN.

## Changes

- Change some error logs to warnings

## DevOps Notes

- [x] This PR does not require configuration changes (e.g., environment variables, GitHub secrets, VM resources)
- [x] This PR does not affect public APIs
- [x] This PR does not rely on a new version of external services (PoET, elasticsearch, etc.)
- [ ] This PR does not make changes to log messages (which monitoring infrastructure may rely on)

### CODE REVIEW

File: activation/atxdb.go [hunk 1]
```diff
@@ -186,11 +186,14 @@ func (db *DB) ProcessAtx(ctx context.Context, atx *types.ActivationTx) error {
...
 	if err := db.ContextuallyValidateAtx(atx.ActivationTxHeader); err != nil {
-		db.log.WithContext(ctx).With().Error("atx failed contextual validation",
+		db.log.WithContext(ctx).With().Warning("atx failed contextual validation",
 			atx.ID(),
 			log.FieldNamed("atx_node_id", atx.NodeID),
 			log.Err(err))
 		// TODO: Blacklist this miner
+		// NOTE(nkryuchkov): Sometimes, this case may be seen in devnet logs
+		// with error: "last atx is not the one referenced",
+		// perhaps we need to add a threshold for blacklisting.
```
Comment (dshulyak) [inline activation/atxdb.go]: it is very likely due to the bug in the codebase https://github.com/spacemeshos/go-spacemesh/issues/2871 . i don't think that these todo's about blacklisting should be implemented until we know that software always works as intended.

right now blacklisting will do more harm than good. i would just remove todo and note, i think it is more confusing then useful

**Review by dshulyak — APPROVED** [review]

File: activation/atxdb.go (hunk 1 above)
Comment (nkryuchkov) [reply activation/atxdb.go]: I think removing only this to-do won't give any benefits as we have similar to-do's in the codebase. Maybe something like `TODO: Consider blacklisting after we ensure that this case always works as intended (see #2871)`?

Comment (dshulyak) [reply activation/atxdb.go]: i don't think that this todo make any sense. you can't just go and fix this todo, it needs to be designed properly for the whole project. in my opinion you just added more unnecessary comments to an already confusing todo, that's why i suggested to remove it altogether

Comment (nkryuchkov) [reply activation/atxdb.go]: > you can't just go and fix this todo, it needs to be designed properly for the whole project

Well, that's true. I'll remove it then

**Comment by nkryuchkov** [comment]
bors merge

**Pull request closed without merging**